sklearn/covariance/tests/test_robust_covariance.py

```diff
@@ -39,6 +41,10 @@ def test_mcd():
     # 1D data set
     launch_mcd_on_dataset(500, 1, 100, 0.001, 0.001, 350)
 
+def test_fast_mcd():
+    X = np.arange(100)
+    assert_raise_message(ValueError, 'expects at least 2', fast_mcd, X)
```

Comment:
'expects at least 2 samples' would be more precise (e.g. it could be features instead of samples).